### ISSUE TITLE
more_page invalid key error when using parsed data

### ISSUE THREAD

**Issue by bulburDE** [dwainscheeren/dwains-lovelace-dashboard#290]
I can't get it working to parse data from the more_page.yaml into the page.yaml. In the [documentation](https://dwainscheeren.github.io/dwains-lovelace-dashboard/addons/more_page.html) the option is described but I couldn't find any examples specifically for the more_page. 

This is my more_page.yaml:
```
# https://dwainscheeren.github.io/dwains-lovelace-dashboard/configuration/more_page.html

more_page:
  addons:
    - name: System Stats
      icon: mdi:chart-bar
      path: 'dwains-dashboard/addons/more_page/system_stats/page.yaml'
      data:
        name: Test
```

This is `dwains-dashboard/addons/more_page/system_stats/page.yaml`:
```
- type: custom:dwains-flexbox-card
  items_classes: 'col-xs-12 col-sm-6 col-sm-6'
  cards:
    - type: entities
      entities:
        - entity: sensor.glances_ha_cpu_used
          group: true
          name: {{ name }}
```
When I reload the theme I get the following error message:
```
2021-05-03 22:54:54 ERROR (SyncWorker_10) [custom_components.dwains_dashboard.process_yaml] invalid key: "OrderedDict([('name', None)])"
in "/config/dwains-dashboard/addons/more_page/system_stats/page.yaml", line 8, column 0
```

**Comment by AalianKhan** [comment]
Hey,
Since you are pulling information from the data tag, I think you will have to configure it like this
```YAML
- type: custom:dwains-flexbox-card
  items_classes: 'col-xs-12 col-sm-6 col-sm-6'
  cards:
    - type: entities
      entities:
        - entity: sensor.glances_ha_cpu_used
          group: true
          name: {{ (data | fromjson)['name'] }}
```
Hope this helps

**Comment by dwainscheeren** [comment]
Also add `# dwains_dashboard`to the first line of page.yaml

**Comment by bulburDE** [comment]
@AalianKhan thanks, that's the right syntax for getting information from data. I've tried a lot, but couldn't get any since I was missing the first line @dwainscheeren mentioned.

I've looked for it in the addons repo and it's missing in most addons. A hint in the documentation would be helpful, but I think more significant is the absence of the examples that are referenced in the documentation. All examples that are referenced there are missing, e.g. `dwains-dashboard/addons/rooms/hello-room`, `dwains-dashboard/addons/more_page/statistics/`, `dwains-dashboard/addons/persons/hello-person/`
So when starting I just picked a random one to build up from but that was missing the first line.
